README.md

```diff
@@ -5,14 +5,12 @@ Previous versions (< 3.0.0) implemented the v0 proposal, which was only implemen
 
 ## Usage
 
-Import `register` and call it with your component, a tag name<strong>*</strong>, and a list of attribute names you want to observe:
+Import `register` and call it with your component, a tag name<strong>\*</strong>, and a list of attribute names you want to observe:
```

Comment:
Sorry, but can we revert this whole file? I understand it's formatting changes done by the precommit hook due to the original readme changes, but I'd prefer to land this w/ that.

Reply:
Sure, but shouldn't we fix the issue at its root for follow-up PRs?

Reply:
Alternatively, sure, can format the entire project separately.

Reply:
That would be great 👍🏽  Could you take care of this?